### PR TITLE
Allow non-aligned PSTR()

### PR DESCRIPTION
**Pull request by s-hadinger**
Report PR esp8266/Arduino#7275, but preserved the `__pstr__`

### CODE REVIEW

**Review by earlephilhower — APPROVED** [review]
Thx

**Comment by earlephilhower** [comment]
I'll wait until the main Arduino PR gets merged in case we do something different for the UMM_INFO debug stuff.

**Review by earlephilhower — CHANGES_REQUESTED** [review]
Once the main core is approved again, please port that here so it doesn't get lost.  Thx!

**Comment by s-hadinger** [comment]
Change from main core is now reported here.

**Review by earlephilhower — APPROVED** [review]

**Pull request merged**
